scripts/install-vm.sh

```diff
@@ -88,4 +88,36 @@ EOF
 $SUDO groupadd -r nucliadb
 $SUDO useradd -M -r -g nucliadb nucliadb
 $SUDO mkdir /var/lib/nucliadb
-$SUDO chown nucliadb:nucliadb /var/lib/nucliadb
\ No newline at end of file
+$SUDO chown nucliadb:nucliadb /var/lib/nucliadb
+
+# Install upgrade script
+cat << EOF | $SUDO tee /usr/bin/upgrade-nucliadb > /dev/null
+#!/bin/bash
+#
+# The purpose of this script is to be used to upgrade NucliaDB to the latest
+# version. Works if it has been installed using the install-vm.sh script.
+#
+set -e
+INSTALL_DIR=/opt/nucliadb
+
+SUDO=''
+if (($EUID != 0)); then
+  SUDO='sudo'
+fi
+
+$SUDO systemctl stop nucliadb
+$SUDO $INSTALL_DIR/bin/pip install --upgrade nucliadb
```

Comment:
WDYT about supporting specific version to upgrade to?
Otherwise this will pull latest, and we may not want this in all cases.